.github/workflows/objc_cocoapods.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         PLATFORM: ["ios", "macos", "tvos"]
         CONFIGURATION: ["Debug", "Release"]
     steps:
-    - uses: actions/checkout@v3
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
```

Comment:
Why the custom commit?

Reply:
For better consistency with [go/github-docs/security-recommendations#auditfuture-practices-5](http://go/github-docs/security-recommendations#auditfuture-practices-5).

Note: this PR is a WIP, I'll be converting it to multiple cls once it's ready :P

Reply:
That page item goes on to say:

> GitHub-owned actions (their name starts with actions/ or github/) may be pinned by version instead of hash.

It's only the third_party ones they seem to suggest pinning on.  Anyway, up to you on it.

> Note: this PR is a WIP, I'll be converting it to multiple cls once it's ready :P

Ack, you manually added me, so I thought I was supposed to look.  :)